samples/appendingEvents.go

```diff
@@ -0,0 +1,142 @@
+package samples
```

Comment:
Ad. file naming: the general convention is that you should create one-word file names. But sometimes it's not possible, but in that scenario snake case is preferred over a camel case. 

Good example: https://github.com/golang/go/tree/master/src/math
![image](https://user-images.githubusercontent.com/5737091/134571440-564befbd-7944-4bae-9933-780f66822c27.png)